app/src/openxr/cpp/OpenXRInput.h

```diff
@@ -36,7 +36,7 @@ class OpenXRInput {
   XrResult Update(const XrFrameState& frameState, XrSpace baseSpace, const vrb::Matrix& head, float offsetY, device::RenderMode renderMode, ControllerDelegate& delegate);
   int32_t GetControllerModelCount() const;
   std::string GetControllerModelName(const int32_t aModelIndex) const;
-  void UpdateInteractionProfile(ControllerDelegate&);
+  void UpdateInteractionProfile(ControllerDelegate&, const char *emulateProfile);
```

Comment:
I don't like very much the change in this API but it's indeed convenient for code reuse. In any case, in order to make it a bit less ugly, let's use a default value `const char* emulatedProfile = nullptr` (also * goes with the type) so that we don't have to pass `nullptr` in some call sites.

Reply:
yeah, should have added the default value the first time. Good catch!